.github/workflows/main.yml

```diff
@@ -1,8 +1,10 @@
 name: noop run
 
 on:
-  - pull_request
-  - push
+  pull_request: {}
+  push:
+    branches:
+      - master
```

Comment:
would it make sense to cover `main` too?

Reply:
none of those repos currently have a main branch. I don't think it's required to list it here.

Reply:
lol. turns out puppet-lint-lookup_in_parameter-check is quite new and uses main. I updated the action.

Reply:
Would it make sense to change the default branch in the single instance rather than sync it to many places?

I also wonder if modulesync somehow has the information of the default branch.

And as a last note: you changed it in the test for this repo, not in the moduleroot.

Reply:
fixed the github actions. I'm always unsure about changing default branch names and causing potential harm. If we can avoid this by adding a single line to the github action I think that's fine?

Reply:
I had a look, but modulesync doesn't expose the information, even though it knows about it.

Any thought about making it a parameter instead of hardcoding it? Then we can solve it via `.sync.yml` in the one exception.

Reply:
two things:
* this file is for the puppet-lint_modulesync_configs repo itself
* The file below, moduleroot/.github/workflows/test.yml.erb, is for the plugins, I updated it to run on merges to master and main. We don't need to make it configureable. We can just list all branch names.